src/Features/Core/Portable/AddImport/SymbolReferenceFinder_PackageAssemblySearch.cs

```diff
@@ -154,6 +142,11 @@ private partial class SymbolReferenceFinder
                 {
                     cancellationToken.ThrowIfCancellationRequested();
                     var compilation = await project.GetCompilationAsync(cancellationToken).ConfigureAwait(false);
+                    if (compilation == null)
```

Comment:
how can this happen?

Reply:
The project may not support compilation.

Reply:
How would that happen with SymbolReferenceFinder?  SymbolReferenceFinder is only for the AddImport feature, which is only for C#/VB.  So it should always support compilations.  If it didn't we would have crashed on hte line below.

Reply:
Indeed, we would have crashed. I have enabled nullable reference checks and the compiler told me we would have crashed. If it is not possible to get here for non-C#/VB projects then I'll change this to `RoslynDebug.Assert(compilation != null);`. I wasn't sure that's the case because it's not clear to me that _SymbolReferenceFinder is only for the AddImport feature_.

Reply:
> Indeed, we would have crashed. I have enabled nullable reference checks and the compiler told me we would have crashed.

The compiler is incorrect.  it doesn't know how to associate the idea of "we are in C#/VB context only, so GetCompilationAsync will always succeed".  This is a limitation or NRT.  One thing we've done is add methods like `GetRequiredCompilationAsync` to have the code more clearly state its intent here to satisfy the NRT system.

Reply:
> I wasn't sure that's the case because it's not clear to me that SymbolReferenceFinder is only for the AddImport feature.

It's nested type within of `AbstractAddImportFeatureService` which is in the `AddImport` namespace :)

Reply:
I'm not familiar with the implementation of this feature, so I wasn't sure if this particular class is not looking into other projects as well to determine something that's needed for AddImport. As I said, I have no problem changing this to `RoslynDebug.Assert(compilation != null);` to help the compiler out.